duckduckgo/iOS
# Feature/xcode9 fixes pt2

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/452112709942758
CC:

**Description**:

Custom navigation bar and management of swipe to hide/show bars.  Gives the bars a consistent look and feel across all devices and orientations from which we can build upon moving forward.

**Steps to test this PR**:
1. On a range of devices and orientations, perform searches and visit some websites.
1. Scroll content to hide/show bars
1. Use the fire button to return to home screen (bars should be as expected)
1. Run the Screenshots target as tests (Cmd-U)
1. Run the Integration tests target as tests (Cmd-U)

###### Reviewer Checklist:


###### PR DRI Checklist:
